src/cosmos/Swap.fixture.tsx

```diff
@@ -15,7 +15,7 @@ import { useCallback, useEffect, useState } from 'react'
 import { useValue } from 'react-cosmos/fixture'
 import { Field } from 'state/swap'
 import styled from 'styled-components/macro'
-import { H2, H3 } from 'theme/type'
+import * as Type from 'theme/type'
```

Comment:
I hadn't realized H2/H3 came from here - it's fine to import them directly if you want to revert that.